File: backend/src/main/java/com/neptune/itcenter/boms/Invoice.java
package com.neptune.itcenter.boms;

public class Invoice extends Bom {
    private User student;
    private Integer total;
    private boolean confirmed;

    public Integer getTotal() {
        return total;
    }

    public void setTotal(Integer total) {
        this.total = total;
    }

    public User getStudent() {
        return student;
    }

    public void setStudent(User student) {
        this.student = student;
    }

    public boolean isConfirmed() {
        return confirmed;
    }

    public void setConfirmed(boolean confirmed) {
        this.confirmed = confirmed;
    }
}

File: frontend/src/app/models/invoice.ts
import { Timestamp } from './timestamp';
import { User } from './user';

export class Invoice extends Timestamp {
  id: number;
  total: number;
  student: User;
  confirmed: boolean;
  loading = false;
  cancelLoading = false;

  constructor(obj?) {
    super(obj);
    if (obj) {
      this.id = obj.id;
    }
  }
}
